src/Auth_Command.php

```diff
@@ -546,14 +577,17 @@ private function get_auths( $site_url, $user, $error_if_empty = true ) {
 	 * ## OPTIONS
 	 *
 	 * [<site-name>]
-	 * : Name of website / `global` for global scope.
+	 * : Name of website / `global` for global scope / `admin-tools` for admin-tools.
 	 *
 	 * [--user=<user>]
 	 * : Username that needs to be deleted.
 	 *
 	 * [--ip]
 	 * : IP to remove. Default removes all.
-	 *
+	 * 
+	 * [--y]
```

Comment:
Rename this falg to `--yes`. Then you won't need to check if flag is present as `EE::confirm` has the functionality to handle it.

Reply:
Fixed, @kirtangajjar. Now, it works with `--yes` flag instead of `--y`.